### PR TITLE
Remove outdated links and add the link to the Rizin blog

### PR DESCRIPTION
**Pull request by h4x5p4c3**
1. removed cheatsheet by pwntester.
2. removed rizin wike which isn't actually present 
3. changed `https://rizin.today` to `https://rizin.re/posts/`

### CODE REVIEW

**Comment by h4x5p4c3** [comment]
*rizin wiki

**Comment by officialcjunior** [comment]
Great, thanks.
I think, the first paragraph of the page can also be removed/tweaked, since the author could not have used Rizin in 2011.

**Comment by h4x5p4c3** [comment]
welcome, ill remove it then

File: src/crackmes/avatao/01-reverse4/rizin.md [hunk 1]
```diff
@@ -71,7 +71,5 @@ use the git version!
 
 Some highly recommended reading materials:
 
-- [Cheatsheet by pwntester](https://github.com/pwntester/cheatsheets/blob/master/rizin.md)
 - [Rizin Book](https://www.gitbook.com/book/rizinorg/rizinbook/details)
```
Comment (wargio) [inline src/crackmes/avatao/01-reverse4/rizin.md:74]: point this to `book.rizin.re`
```suggestion
- [Rizin Book](https://book.rizin.re)
```

**Review by wargio — CHANGES_REQUESTED** [review]

**Review by wargio — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**